loops.js

```diff
@@ -30,9 +42,17 @@
 // .... etc.
 // 15 is oneven
 
-// Tip: een getal is oneven als je het door 2 deelt en er blijft nog iets over. Een getal is even als je het deelt door 2 en er blijft niets over..
+// Tip: een getal is oneven als je het door 2 deelt en er blijft nog iets over. Een getal is even als je het deelt door 2 en er blijft niets over..(bruh)
 // ==========================================
-
+console.log("====================Opdracht 3======================");
+for (let k = 0; k < 15; k++) {
+    let remainder = k % 2;
+    if (remainder === 0) {
```

Comment:
could have been a single line with a ternary comparison

Reply:
Zo bijvoorbeeld:
`    if (i % 2 === 0) { `

Reply:
bedankt voor je comments, ik dacht zelf hier aan;
console.log(k % 2 === 0 ? 'is even' : 'is oneven'); 
dan heb je alles in in :)

Reply:
en dan nog de k er bij 😆